### ISSUE TITLE
Multiple custom units

### ISSUE THREAD

**Issue by baron1405** [baron1405/meazure#1]
Nice program! It is already proving very useful.

It would be helpful to be able to add numerous different custom units to the units menu, rather than just one. I am measuring digital photomicrographs that might be taken with any of four magnifications, and might be displayed at any of several window sizes, so it would be handy to be able to set up custom units for each, and then pick them from the units menu as needed.

If you were to implement this, it would also help to have more characters to name these units (maybe up to 30, for names like "microns 400x 1280x1024 preview").

I hope that is a useful suggestion.

**Comment by DonpK** [comment]
I agree, having multiple custom units would be helpful. This is an old enhancement issue, so maybe it's been added to Ver. 4.0 which I just downloaded. 

Thanks for a great app which I've used regularly for years and support regularly on PayPal
